frontend/src/Dashboard.tsx

```diff
@@ -245,19 +247,11 @@ const Dashboard = () => {
                       className="border border-gray-300 mb-4 p-4 rounded-md"
                     >
                       <div>
-                        <p className="font-bold mb-2">Pet Name : {myApplications.pet.name}</p>
-                        <p>
-                          Location: {myApplications?.location?.address ?? ""},{" "}
-                          {myApplications?.location?.city ?? ""},{" "}
-                          {myApplications?.location?.zipcode ?? ""}
-                        </p>
-                        <p>Pay: ${myApplications.job.pay}</p>
-                        <p>Start: {myApplications.job.start}</p>
-                        <p>End: {myApplications.job.end}</p>
-                        <p className="font-bold mb-2">
-                          Application Status:{" "}
-                          {!myApplications.status ? "No Decision" : myApplications.status}
-                        </p>
```

Comment:
Please revert to changes present in develop branch for "locations" and  "application status"